.bazelrc

```diff
@@ -13,5 +13,5 @@ build --define tsl_protobuf_header_only=true
 build --define=use_fast_cpp_protos=true
 build --define=allow_oversize_protos=true
 
-build -c opt
```

Comment:
@avik-pal this file shouldn't have been merged, I'm going to undo this

Reply:
oops yeah accidentally committed it. Is there as way to do debug build without updating the bazelrc?

Reply:
yeah bazel build -c dbg //:enzymexlamlir-opt